packages/styling/src/styles/scheme.test.ts

```diff
@@ -1,12 +1,10 @@
 import { getThemedContext } from './scheme';
 import { createTheme } from './theme';
-import { IPartialTheme, ISchemeNames, ITheme } from '../interfaces/index';
+import { ISchemeNames, ITheme } from '../interfaces/index';
 import { loadTheme } from './theme';
-import { Customizations, mergeCustomizations, ICustomizerContext } from '@uifabric/utilities';
+import { ICustomizerContext } from '@uifabric/utilities';
```

Comment:
unused imports.

Reply:
Shouldn't we have some kind of lint rule (or compile error) catching these?

Reply:
I honestly am not sure why this wasn't caught. Maybe linting disabled for .test files?